test/unit/filters/shortUrlFilterSpec.js

```diff
@@ -0,0 +1,59 @@
+/* global describe, it, inject, expect, beforeEach */
+
+describe('shortUrl filter', function () {
+  var $filter, shortUrlFilter
```

Comment:
Today I've learned that you can share variables by assigning them to `this`.

See above for explanation.